features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/event/TimelineItemFileView.kt

```diff
@@ -57,16 +58,23 @@ fun TimelineItemFileView(
             contentAlignment = Alignment.Center,
         ) {
             Icon(
-                imageVector = Icons.Filled.Attachment,
+                modifier = Modifier.size(20.dp),
```

Comment:
I actually made some changes to this icon in [this PR](https://github.com/vector-im/element-x-android/pull/498/files#diff-38b8312dd1cb2ca03c6bd5ef39fa62b33a4adc7dd3196a5ee7ec9af1442cf879R64) (I rotated the icon 45º) to make it match the icon used in the 'message preview' in the action list. I didn't realise you still had some changes to do in this component, sorry.

Reply:
Yes I've seen the conflict, just fixing.